### ISSUE TITLE
Delete deprecated fields from Land Library resource page

### ISSUE THREAD

**Issue by lisettemeij** [landportal/drupal#81]
Can we delete the deprecated fields from the Land Library resource view? I'm particularly thinking about the 'Country' field.

It's getting ridiculous with all the double country names appearing in the resource view and I can't seem to delete them, it keeps showing back up after I've deleted it:

https://landportal.info/library/resources/strategic-2010/strategic-citywide-spatial-planning-situational-analysis

We paid GLTN to make this resource view good and it's just annoying that this keeps showing up.

**Comment by lisettemeij** [comment]
Could we give this a high priority please? It seems like a really simple thing to improve + would look A LOT better once it's done.

Carlos was asking if he can look with Jules while he's changing this in code, so he can fix small things like this in the future as well.

**Comment by lisettemeij** [comment]
I did it in the Drupal menu myself, I disabled the Country field. I hope this is ok in the code, but I was just getting fed up with it showing up. Sorry :(

**Comment by jcr** [comment]
The (small) issue is that information is not necessarily duplicated (some countries may appear in both but in some case there is only the information in country and not in the region taxonomy).
While this is still a work in progress (the last step taken was the cleaning of the countries taxonomy) it will only be fully solved by implementing higher geo-standards.

it is OK as long as you only hide it. If you remove completely a field, we loose all the information associated to it.

In short, the next actions (that was in my country/region migration plan) would be to copy all the links between LLR and countries and put them in the geographical focus (aka region) taxonomy.
Step after that, deploy geo field and map the terms in geographical focus to their WKT location so they have proper geo attributes.

**Comment by jcr** [comment]
countries shouldn't be shown anywhere. This taxonomy is definitely deprecated and will be removed from LLR very soon.
